viewing_party/party.py

```diff
@@ -1,23 +1,132 @@
+import statistics
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
-
+# -----------------------------------------
 def create_movie(title, genre, rating):
-    pass
+    # create movie information
+    movie_info = {}
+    if title == None or genre == None or rating == None:
+        return None
+    movie_info['title'] = title
+    movie_info['genre'] = genre
+    movie_info['rating'] = rating
+    return movie_info 

```

Comment:
Instead of creating an empty dictionary, you can build a dictionary literal an example could look like 
```suggestion
if title and genre and rating:
        movie =  {
            "title": title,
            "genre": genre,
            "rating": rating
            }
        return movie
```